lib/jdomodels/src/main/resources/schema/AsynchJobStatus-ddl.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE IF NOT EXISTS `ASYNCH_JOB_STATUS` (
+  `JOB_ID` BIGINT NOT NULL,
+  `ETAG` VARCHAR(256) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci NOT NULL,
+  `JOB_STATE` ENUM('PROCESSING','FAILED','COMPLETE') NOT NULL,
+  `JOB_TYPE` ENUM('TABLE_UPDATE_TRANSACTION','UPLOAD_CSV_TO_TABLE_PREVIEW','DOWNLOAD_CSV_FROM_TABLE','QUERY','QUERY_NEXT_PAGE','BULK_FILE_DOWNLOAD','MIGRATION','DOI','ADD_FILES_TO_DOWNLOAD_LIST','STORAGE_REPORT','JSON_SCHEMA_CREATE','VIEW_COLUMN_MODEL_REQUEST','GET_VALIDATION_SCHEMA','QUERY_DOWNLOAD_LIST', 'ADD_TO_DOWNLOAD_LIST', 'DOWNLOAD_LIST_PACKAGE') NOT NULL,
+  `CANCELING` BIT(1) NOT NULL,
+  `EXCEPTION` VARCHAR(512) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
+  `ERROR_MESSAGE` VARCHAR(3000) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
+  `ERROR_DETAILS` MEDIUMTEXT,
+  `PROGRESS_CURRENT` BIGINT DEFAULT NULL,
+  `PROGRESS_TOTAL` BIGINT DEFAULT NULL,
+  `PROGRESS_MESSAGE` VARCHAR(3000) CHARACTER SET utf8mb4 COLLATE utf8mb4_0900_ai_ci DEFAULT NULL,
+  `STARTED_ON` BIGINT NOT NULL,
+  `STARTED_BY` BIGINT NOT NULL,
```

Comment:
timestamp(3)
this will generate the correct runtime
UPDATE `SOMETHING` SET RUNTIME = UNIX_TIMESTAMP(NOW(3))*1000 - UNIX_TIMESTAMP(UPDATED_ON)*1000 WHERE ID = 1;